src/Kestrel.Core/Internal/Http/HttpParser.cs

```diff
@@ -52,7 +50,7 @@ public unsafe bool ParseRequestLine(TRequestHandler handler, ReadOnlySequence<by
                 // No request line end
                 return false;
             }
-            else if (TryGetNewLine(ref buffer, out var found))
+            else if (TryGetNewLine(in buffer, out var found))
```

Comment:
Do we need `in` here? (I heard this would work without adding in on the call site.)

Reply:
Nope, will remove

Reply:
Done